<!doctype html><html lang=en><head><meta charset=utf-8><meta name=description content="Past Data Analyses  These are mostly from the genetics lab I work at
  Spheroid Analysis ![[notes/images/inner_outer_seg.gif]]
 Example plot from one of my EDAs"><title>Daniel George</title><meta name=viewport content="width=device-width,initial-scale=1"><link rel="shortcut icon" type=image/png href=https://dangeorge.me/icon.png><link href=https://dangeorge.me/styles.547fb332ce56cdc523146ba5ede80d7f.min.css rel=stylesheet><link href=https://dangeorge.me/styles/_light_syntax.86a48a52faebeaaf42158b72922b1c90.min.css rel=stylesheet id=theme-link><script src=https://dangeorge.me/js/darkmode.3e6c4a742e821ec97a449587b42f6b58.min.js></script>
<script src=https://dangeorge.me/js/util.6f22941e242efae60fd84e7c32e874fa.min.js></script>
<link rel=preload href=https://cdn.jsdelivr.net/npm/katex@0.15.1/dist/katex.min.css as=style onload='this.onload=null,this.rel="stylesheet"' integrity=sha384-R4558gYOUz8mP9YWpZJjofhk+zx0AS11p36HnD2ZKj/6JR5z27gSSULCNHIRReVs crossorigin=anonymous><script defer src=https://cdn.jsdelivr.net/npm/katex@0.15.1/dist/katex.min.js integrity=sha384-z1fJDqw8ZApjGO3/unPWUPsIymfsJmyrDVWC8Tv/a1HeOtGmkwNd/7xUS0Xcnvsx crossorigin=anonymous></script>
<script defer src=https://cdn.jsdelivr.net/npm/katex@0.15.1/dist/contrib/auto-render.min.js integrity=sha384-+XBljXPPiv+OzfbB3cVmLHf4hdUFHlWNZN5spNQ7rmHTXpd7WvJum6fIACpNNfIR crossorigin=anonymous></script>
<script defer src=https://cdn.jsdelivr.net/npm/katex@0.16.2/dist/contrib/copy-tex.min.js integrity=sha384-ww/583aHhxWkz5DEVn6OKtNiIaLi2iBRNZXfJRiY1Ai7tnJ9UXpEsyvOITVpTl4A crossorigin=anonymous></script>
<script async src=https://unpkg.com/@floating-ui/core@0.7.3></script>
<script async src=https://unpkg.com/@floating-ui/dom@0.5.4></script>
<script async src=https://dangeorge.me/js/popover.f03552ccb84d99ca615d1cfb9abde59e.min.js></script>
<script defer src=https://dangeorge.me/js/code-title.ce4a43f09239a9efb48fee342e8ef2df.min.js></script>
<script defer src=https://dangeorge.me/js/clipboard.2913da76d3cb21c5deaa4bae7da38c9f.min.js></script>
<script defer src=https://dangeorge.me/js/callouts.7723cac461d613d118ee8bb8216b9838.min.js></script>
<script>const SEARCH_ENABLED=!1,LATEX_ENABLED=!0,PRODUCTION=!0,BASE_URL="https://dangeorge.me",fetchData=Promise.all([fetch("https://dangeorge.me/indices/linkIndex.d67d35aaafb37e4d28fe92a3bdcbd606.min.json").then(e=>e.json()).then(e=>({index:e.index,links:e.links})),fetch("https://dangeorge.me/indices/contentIndex.ca0573de7190651989aabdbe2ca01541.min.json").then(e=>e.json())]).then(([{index:e,links:t},n])=>({index:e,links:t,content:n})),render=()=>{const e=new URL(BASE_URL),t=e.pathname,n=window.location.pathname,o=t==n;addCopyButtons(),addTitleToCodeBlocks(),addCollapsibleCallouts(),initPopover("https://dangeorge.me",!0);var s=document.getElementsByClassName("mermaid");s.length>0&&import("https://unpkg.com/mermaid@9/dist/mermaid.esm.min.mjs").then(e=>{e.default.init()})},init=(e=document)=>{addCopyButtons(),addTitleToCodeBlocks(),renderMathInElement(e.body,{delimiters:[{left:"$$",right:"$$",display:!0},{left:"$",right:"$",display:!1}],macros:{'’':"'"},throwOnError:!1})}</script><script type=module>
    import { attachSPARouting } from "https:\/\/dangeorge.me\/js\/router.9d4974281069e9ebb189f642ae1e3ca2.min.js"
    attachSPARouting(init, render)
  </script></head><script async src="https://www.googletagmanager.com/gtag/js?id=G-XYFD95KB4J"></script>
<script>var doNotTrack=!1;if(!doNotTrack){window.dataLayer=window.dataLayer||[];function gtag(){dataLayer.push(arguments)}gtag("js",new Date),gtag("config","G-XYFD95KB4J",{anonymize_ip:!1})}</script><body><div id=search-container><div id=search-space><input autocomplete=off id=search-bar name=search type=text aria-label=Search placeholder="Search for something..."><div id=results-container></div></div></div><script src=https://cdn.jsdelivr.net/npm/flexsearch@0.7.21/dist/flexsearch.bundle.js integrity="sha256-i3A0NZGkhsKjVMzFxv3ksk0DZh3aXqu0l49Bbh0MdjE=" crossorigin=anonymous defer></script>
<script defer src=https://dangeorge.me/js/full-text-search.e6e2e0c213187ca0c703d6e2c7a77fcd.min.js></script><div class=singlePage><header><h1 id=page-title><a href=https://dangeorge.me>Daniel George</a></h1><div class=spacer></div><div id=search-icon><p>Search</p><svg tabindex="0" aria-labelledby="title desc" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 19.9 19.7"><title id="title">Search Icon</title><desc id="desc">Icon to open search</desc><g class="search-path" fill="none"><path stroke-linecap="square" d="M18.5 18.3l-5.4-5.4"/><circle cx="8" cy="8" r="7"/></g></svg></div><div class=darkmode><input class=toggle id=darkmode-toggle type=checkbox tabindex=-1>
<label id=toggle-label-light for=darkmode-toggle tabindex=-1><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" id="dayIcon" viewBox="0 0 35 35" style="enable-background:new 0 0 35 35"><title>Light Mode</title><path d="M6 17.5C6 16.672 5.328 16 4.5 16h-3C.672 16 0 16.672.0 17.5S.672 19 1.5 19h3C5.328 19 6 18.328 6 17.5zM7.5 26c-.414.0-.789.168-1.061.439l-2 2C4.168 28.711 4 29.086 4 29.5 4 30.328 4.671 31 5.5 31c.414.0.789-.168 1.06-.44l2-2C8.832 28.289 9 27.914 9 27.5 9 26.672 8.329 26 7.5 26zm10-20C18.329 6 19 5.328 19 4.5v-3C19 .672 18.329.0 17.5.0S16 .672 16 1.5v3C16 5.328 16.671 6 17.5 6zm10 3c.414.0.789-.168 1.06-.439l2-2C30.832 6.289 31 5.914 31 5.5 31 4.672 30.329 4 29.5 4c-.414.0-.789.168-1.061.44l-2 2C26.168 6.711 26 7.086 26 7.5 26 8.328 26.671 9 27.5 9zM6.439 8.561C6.711 8.832 7.086 9 7.5 9 8.328 9 9 8.328 9 7.5c0-.414-.168-.789-.439-1.061l-2-2C6.289 4.168 5.914 4 5.5 4 4.672 4 4 4.672 4 5.5c0 .414.168.789.439 1.06l2 2.001zM33.5 16h-3c-.828.0-1.5.672-1.5 1.5s.672 1.5 1.5 1.5h3c.828.0 1.5-.672 1.5-1.5S34.328 16 33.5 16zM28.561 26.439C28.289 26.168 27.914 26 27.5 26c-.828.0-1.5.672-1.5 1.5.0.414.168.789.439 1.06l2 2C28.711 30.832 29.086 31 29.5 31c.828.0 1.5-.672 1.5-1.5.0-.414-.168-.789-.439-1.061l-2-2zM17.5 29c-.829.0-1.5.672-1.5 1.5v3c0 .828.671 1.5 1.5 1.5s1.5-.672 1.5-1.5v-3C19 29.672 18.329 29 17.5 29zm0-22C11.71 7 7 11.71 7 17.5S11.71 28 17.5 28 28 23.29 28 17.5 23.29 7 17.5 7zm0 18c-4.136.0-7.5-3.364-7.5-7.5s3.364-7.5 7.5-7.5 7.5 3.364 7.5 7.5S21.636 25 17.5 25z"/></svg></label><label id=toggle-label-dark for=darkmode-toggle tabindex=-1><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" id="nightIcon" viewBox="0 0 100 100" style="enable-background='new 0 0 100 100'"><title>Dark Mode</title><path d="M96.76 66.458c-.853-.852-2.15-1.064-3.23-.534-6.063 2.991-12.858 4.571-19.655 4.571C62.022 70.495 50.88 65.88 42.5 57.5 29.043 44.043 25.658 23.536 34.076 6.47c.532-1.08.318-2.379-.534-3.23-.851-.852-2.15-1.064-3.23-.534-4.918 2.427-9.375 5.619-13.246 9.491-9.447 9.447-14.65 22.008-14.65 35.369.0 13.36 5.203 25.921 14.65 35.368s22.008 14.65 35.368 14.65c13.361.0 25.921-5.203 35.369-14.65 3.872-3.871 7.064-8.328 9.491-13.246C97.826 68.608 97.611 67.309 96.76 66.458z"/></svg></label></div></header><article><p class=meta>Last updated
Feb 10, 2023
<a href=https://github.com/degtrdg/blog/tree/hugo/content/portfolio/Portfolio.md rel=noopener>Edit Source</a></p><ul class=tags></ul><aside class=mainTOC><details><summary>Table of Contents</summary><nav id=TableOfContents><ol><li><a href=#past-data-analyses>Past Data Analyses</a><ol><li><a href=#spheroid-analysis>Spheroid Analysis</a></li><li><a href=#physically-unclonable-function-puf-pipeline-and-analysis>Physically Unclonable Function (PUF) Pipeline and Analysis</a></li><li><a href=#finding-genomic-safe-harbors-for-cho>Finding Genomic Safe Harbors for CHO</a></li></ol></li><li><a href=#projects>Projects</a><ol><li><a href=#small-molecule-autocomplete-rnn>Small Molecule Autocomplete RNN</a></li></ol></li></ol></nav></details></aside><a href=#past-data-analyses><h2 id=past-data-analyses><span class=hanchor arialabel=Anchor># </span>Past Data Analyses</h2></a><blockquote><p>These are mostly from the genetics lab I work at</p></blockquote><hr><a href=#spheroid-analysis><h3 id=spheroid-analysis><span class=hanchor arialabel=Anchor># </span>Spheroid Analysis</h3></a><p><img src=/notes/images/inner_outer_seg.gif width=auto></p><blockquote><p>Example plot from one of my EDAs</p></blockquote><p>Spheroids are 3D cultures that mimic tissues and micro tumors better than the 2D cultures we see in Petri dishes for a variety of reasons. When extending biological circuits from the 2D to the more realistic 3D, the morphology of the cells could affect things like protein production. Protein synthesis is an inherently stochastic process, so when creating biological circuits quantifying a measure of
<a href=https://en.wikipedia.org/wiki/Cellular_noise rel=noopener>noise</a> in the system is quite useful. This is extending the 2D
<a href="https://academic.oup.com/nar/article/48/16/9406/5893975?login=false" rel=noopener>analysis</a> to spheroids. You can find out more in my
<a href=https://github.com/degtrdg/spheroid-analysis/tree/main/presentations rel=noopener>slides</a>.</p><p>Links:</p><ul><li><a href=https://github.com/degtrdg/spheroid-analysis/ rel=noopener>Github</a></li></ul><hr><a href=#physically-unclonable-function-puf-pipeline-and-analysis><h3 id=physically-unclonable-function-puf-pipeline-and-analysis><span class=hanchor arialabel=Anchor># </span>Physically Unclonable Function (PUF) Pipeline and Analysis</h3></a><p>PUFs are a type of fingerprint that is introduced in manufacturing to show the lineage of a device. They are mainly used in integrated circuits, but they can also be used in other domains like biology. Using CRISPR one can make a PUF to that is impossible to reproduce and can be used to determine the lineage of a cell line (
<a href=https://pubmed.ncbi.nlm.nih.gov/35507652/ rel=noopener>CRISPR-PUF</a>). I wrote a pipeline to process the millions of DNA sequencing reads that are used to find a &lsquo;distance&rsquo; between cell lines. If they are the same cell line the distance would be small, and it would be large if the cell lines are different.</p><p>Links:</p><ul><li><a href=https://github.com/degtrdg/puf_analysis rel=noopener>Github</a></li></ul><hr><a href=#finding-genomic-safe-harbors-for-cho><h3 id=finding-genomic-safe-harbors-for-cho><span class=hanchor arialabel=Anchor># </span>Finding Genomic Safe Harbors for CHO</h3></a><p><img src="/notes/images/Pasted image 20230210133224.png" width=auto></p><blockquote><p>Logic behind pipeline from
<a href=https://github.com/degtrdg/genomic-safe-harbors-CHO/tree/main/presentation rel=noopener>slides</a></p></blockquote><p>When you edit a cell line with techniques like CRISPR, if you add code without taking into account the function of the regions around it, there might be unintended consequences. There are certain regions that are considered
<a href=https://www.sciencedirect.com/science/article/pii/S2667237521002319 rel=noopener>safe harbors</a> where your changes won&rsquo;t interfere with existing functionality. There was code from a paper which defined requirements for safe harbors for the human genome, but we needed it to work for the Chinese Hamster Ovary, so I wrote a Python port than can be extended to any genome. See
<a href=https://github.com/degtrdg/genomic-safe-harbors-CHO/tree/main/presentation rel=noopener>slides</a> for more info.</p><p>Links:</p><ul><li><a href=https://github.com/degtrdg/genomic-safe-harbors-CHO rel=noopener>Github</a></li></ul><a href=#projects><h2 id=projects><span class=hanchor arialabel=Anchor># </span>Projects</h2></a><hr><a href=#small-molecule-autocomplete-rnn><h3 id=small-molecule-autocomplete-rnn><span class=hanchor arialabel=Anchor># </span>Small Molecule Autocomplete RNN</h3></a><p>Trained LSTM based RNN to have model efficiently enumerate chemical space. Used BFS to take top few possibilities in the probability distribution, made design decisions to make model more creative and incentivized to give shorter outputs.</p><iframe src="https://www.youtube.com/embed/eeZ6qKVMesk?rel=0&modestbranding=1&" width=100% height=300em allowfullscreen></iframe><p>Links:</p><ul><li><a href=https://degtrdg.github.io/chem_autocomplete rel=noopener>Demo</a></li></ul></article><hr><div id=contact_buttons><footer><p>Made by Daniel George using <a href=https://github.com/jackyzha0/quartz>Quartz</a>, © 2023</p><ul><li><a href=https://dangeorge.me>Home</a></li><li><a href=https://github.com/degtrdg>GitHub</a></li></ul></footer></div></div></body></html>